applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp

```diff
@@ -836,7 +836,7 @@ namespace Kratos
 
 		Matrix StiffnessMatrix = ZeroMatrix(msElementSize, msElementSize);
 
-		this->CalculateLeftHandSide(StiffnessMatrix, rCurrentProcessInfo);
+		this->CalculateLumpedMassMatrix(StiffnessMatrix, rCurrentProcessInfo);

```

Comment:
is this wanted?

Reply:
no :(

Reply:
I think you found the reason for the non-working damping problem ..... 👎